### ISSUE TITLE
Destruction Shadowlands quest "Land of Shadows" is missing two NPCs

### ISSUE THREAD

**Issue by Ladgahr** [WarEmu/WarBugs#20902]
**Quest name:** Land of Shadows

**Quest objectives:** "...locate and defeat each of the Shadow Warrior commanders..."
Endhrios Silentfall - 0/1
Celedaine Duskbringer - 0/1
Moroirl Shadowdancer - 0/1

**Expected behavior:**
There should be 3 NPCs present in the highlighted zone

**Actual behavior:**
There is only one of the NPCs present  in the highlighted zone:
Celedaine Duskbringer.

I managed to click on a Moroirl Shadowdancer from a distance when I was standing on a hill where I took the screenshot, but when I got close he wasn't there. Could be stuck in a boulder or something. 

Alternatively, highlighted zone may be showing only the position of Celedaine Duskbringer. There are no other highlighted zones on my map.

**Testing Screenshots/Videos/Evidences (always needed):**

![Popasny_003](https://github.com/WarEmu/WarBugs/assets/133169396/aa4d6cc8-9272-4681-b788-b91162abe4aa)

**Comment by Majorzero2123** [comment]
Hey, can you test this quest again? Endhrios and Mororil should be about here. 20.8k 16.4k 
![image](https://github.com/WarEmu/WarBugs/assets/99697728/3260af5d-9804-4bca-b88b-e1472aefc867)

**Comment by Ladgahr** [comment]
These two are weird so I decided to investigate a little more.
They disappear within 40 seconds after I enter their camp, regardless of me fighting other npcs or not. They simply vanish simultaneosly without moving and if I have them selected, selection clears.

![Popasny_007](https://github.com/WarEmu/WarBugs/assets/133169396/aa724d3d-bfad-4b7a-9269-26db0965157f)

![Popasny_008](https://github.com/WarEmu/WarBugs/assets/133169396/42576fcf-ad5c-48de-a122-2de100886e50)

I tried running around their spot when they were gone, didn't aggro on me.
I tried using /reload. Still not there. I tried waiting for 5 minutes, still gone. I tried running away, then going back - didn't help.

They reappeared when I ran outside of the camp, used /reload, then went back. Tested this a few times. 
~40 seconds after I enter the camp they disappear.

![Popasny_006](https://github.com/WarEmu/WarBugs/assets/133169396/c475ee97-acce-4e88-bd7c-d989221bbc40)


I guess the quest is doable if you know exactly when to find them and can do it fast.

Oh and the third npc, Celedaine, is fine. He's just chilling on his hill, not going anywhere.
